src/styles/searchResults.css

```diff
@@ -0,0 +1,15 @@
+#results-heading {
```

Comment:
Targeting IDs for CSS classes isnt really recommended, they are usually for finding elements with JS and using classnames as the primary method of lookup for CSS. There are a few edge cases but since this isnt one of them, you should replace this with a `.results-heading` class

Reply:
Fair point. classname used now instead